src/tasks/daily.ts

```diff
@@ -349,6 +396,15 @@ export function configureSnojo(): void {
 }
 
 export const DailyTasks: Task[] = [
+  {
+    name: "Chibi Buddy",
+    ready: () =>
```

Comment:
```suggestion
    ready: () => have($item`ChibiBuddy™ (on)`) || have($item`ChibiBuddy™ (off)`),
```
Don't need to check the completion criteria twice.